app/services/solutions.rb

```diff
@@ -0,0 +1,83 @@
+require_relative '../repositories/repository_challenge'
+require_relative './solution'
+
+class Solutions
```

Comment:
`Solution` vs `Solutions` is a subtle and easy to miss difference. Perhaps `Solution` and `Entries` or `Submissions` instead?

Reply:
Nice.
I ll try Submissions && Solution

Reply:
@Hettomei bump on this one. Could we rename this to Submissions?